controller/action/AcquisitionActions.class.php

```diff
@@ -42,13 +42,73 @@ public static function executeYouTube(string $campaignId = '')
 
   public static function executeYT2()
   {
-    return ['acquisition/yt2', ['_no_layout' => true]];
+    return ['acquisition/yt2'];
   }
 
   public static function executeVerify(string $token)
   {
     return ['acquisition/verify', ['token' => $token]];
   }
 
+  public static function executeYoutubeToken()
+  {
+    return ['acquisition/youtube_token', ['_no_layout' => true]];
+  }
+
+  public static function executeYoutubeStatus(string $token)
+  {
+    return ['acquisition/youtube_status', ['token' => $token]];
+  }
+
+  public static function actionYoutubeToken(string $email, string $desired_lbry_channel_name, string $youtube_channel_id){
+
+    $email_is_valid = static::email_verification($email);
+    $desired_lbry_channel_name_is_valid = static::lbry_channel_verification($desired_lbry_channel_name);
```

Comment:
Some inconsistencies here with casing and use of parantheses.

I don't have strong opinions about this, I just like consistency.

Perhaps there is a PHP equivalent of [PrettierJS](https://github.com/prettier/prettier) that could help with this.